status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v10.12.0",
-    "commit-sha1": "f0884d2f51b549fc39226b74d190859485318a0c",
-    "src-sha256": "0qqabvvimmrs2iciq4ld5l9pyziclfjk8gdkp15v2rr5q0k0pgw3"
+    "version": "seanstrom/add-news-notifications-setting",
+    "commit-sha1": "3caeb056d5ffbb81b54f5ac4381e3a76ed770373",
+    "src-sha256": "1cxjrbvji3c5qnkjalmk9p97znryz0s9bkhmcnd78hgfhr8v0h7r"
```

Comment:
I've officially opened the PR for adding the necessary additions to the app settings here: https://github.com/status-im/status-go/pull/6446